### ISSUE TITLE
Correctly normalizing the data for the regression

### ISSUE THREAD

**Issue by Kaushikdey647** [Kaushikdey647/RainfallData#1]
Observe the Regression Model below,  we need a way to normalize the data before it enters the network.
```
model = Sequential(
            [
                Dense(
                    self.__layer1_node,
                    input_dim=self.__inpdim,
                    activation='relu',
                    kernel_initializer=initializers.GlorotNormal(seed=None),
                    bias_initializer='zeros',
                ),
                Dense(
                    self.__layer2_node,
                    activation='sigmoid',
                    kernel_initializer=initializers.HeNormal(seed=None),
                    bias_initializer='zeros',
                ),
                Dense(
                    1,
                    activation='relu',
                    kernel_initializer='normal',
                    bias_initializer='zeros',
                ),
            ]
        )
        model.compile(loss='mean_squared_error',optimizer='adam',metrics=[self.R2Coefficient])
        return model
```
